serverlessworkflow/modules/ROOT/pages/core/defining-an-input-schema-for-workflows.adoc

```diff
@@ -1 +1,58 @@
-//= Defining an input schema for workflows
\ No newline at end of file
+Defining an input schema for workflow
+=====================================
+:compat-mode!:
+
+DataInputSchema is one of the parameters in the Workflow definition as per the Serverless Workflow specification.
```

Comment:
Can you please add a link to the spec where they mention `DataInputSchema`?

See an example: https://raw.githubusercontent.com/kiegroup/kogito-docs/main/serverlessworkflow/modules/ROOT/pages/getting-started/cncf-serverless-workflow-specification-support.adoc

Example:

 ```asciidoc
{spec_doc_url}#workflow-expressions[Serverless Workflow specification]
```

Reply:
Also, can you add an introductory paragraph stating why having a well-defined input schema for the workflow is essential and why users should care about it?